readme.md

```diff
@@ -17,7 +17,19 @@ Using jQuery, write code that makes an AJAX get request to "http://kittengifs.co
 
 Your Answer:
 ```js
-
+$(document).ready(function(){
+  var searchKeyword = function(keyword){
+  var url = http://kittengifs.com/gifs
+  $.ajax({
+    dataType: 'json',
+    url: url,
+    type: 'GET'
+  }).done(function(response){
+    console.log(searchKeyword);
```

Comment:
you want to take advantage of the `response` var, which contains the results here.